patterns/2-structured/service-vs-library.md

```diff
@@ -71,6 +75,8 @@ The likelihood that changes are needed and made in the shared source code
 increases, leading to more frequent opportunities to refine, improve and optimise
 the implementation.
 
+Encourages incremental operational standardisation in release packaging, telemetry, health/readiness endpoints in the shared code.
```

Comment:
Including **why** it encourages that would help readers to understand this even better.

Reply:
Good idea, added in f7af976